Common/Classes/Objects/ProfessionVehicleClassNamesInfo.sqx

```diff
@@ -12,13 +12,19 @@ namespace Intrusion.Common
 	public class ProfessionVehicleClassNamesInfo
 	{
 		// Creates a ProVehicleInfo object.
-		public constructor {
```

Comment:
Always convert these old headers (missing parenthesis) to the new standard when you see them. There are still a few I see... :)